src/wasm.h

```diff
@@ -1610,6 +1614,14 @@ class Module {
     }
     importsMap.erase(name);
   }
+
+  void removeImportsWithSubstring(Name name) {
+    for (int i = imports.size() - 1; i >= 0; i--) {
```

Comment:
That trips a warning because `size_t` (and also `Index`) is unsigned, which makes `i >= 0` always true. Which would be bad.
Alternatively could do:
```c++
for (size_t i = 0; i < imports.size(); ++i) {
  int index = imports.size() - i - 1;
```
though the way it is now feels more direct.
... Although I should probably normalize my `i--`s with my `++i`s